subsys/net/l2/ethernet/bridge/bridge_input.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2025 NXP
```

Comment:
use https://docs.zephyrproject.org/latest/contribute/guidelines.html#copyright-and-license-notices (`SPDX-FileCopyrightText:`)

Reply:
Thanks. Will fix for all new added copyright.